PhysicsTools/PatAlgos/plugins/PATTauProducer.cc

```diff
@@ -386,6 +477,7 @@ void PATTauProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
 	// aTauPFEssential.svCov_ = tauLifetimeInfo.secondaryVertexCov();
       }
     }
+*/
```

Comment:
this commented out block does not match what's in 81X.

also,
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant